### DISCUSSION TITLE
Broken links in documentation

### DISCUSSION THREAD

**Discussion by cnlane** [weather-gov/api#203] [Q&A]
https://www.weather.gov/forecast-icons

Clicking on the two links following "National Center for Atmospheric Reserach"  (Demuth, Lazo, & Morss, 2012; Demuth, Morss, Lazo, & Hilderbrand, 2013)  both lead to

PAGE NOT FOUND
The requested page "/collections/OSGC-000-000-019-084" could not be found.

Also, the page recommends e-mailing icons@noaa.gov, which is no longer a valid e-mail.

**Comment by scadergit** [comment]
Hi, this issue board is only for API-related questions. Please visit https://www.weather.gov/contact and click on "If I have a national level question, who can I contact?" for questions related to content on the weather.gov site.
